.env

```diff
@@ -0,0 +1,15 @@
+# Total amount of tokens to be issued on a new REMME network.
+REMME_TOKEN_SUPPLY=10000000000000
+# The port the REST API server will listen to.
+REMME_REST_API_PORT=8080
+# The port for communications between validators (blockchain protocol).
+REMME_VALIDATOR_PORT=8800
+# The real IP of the validator (required for validator discovery).
+REMME_VALIDATOR_IP=127.0.0.1
+# Peering mode: in `dynamic` mode validator connects to validators listed in
+# `REMME_SEEDS_LIST` and then starts to connect to other validators dynamically
+# building the network. In `static` mode validator connects only to validators
+# listed in `REMME_PEERS_LIST`.
+REMME_PEERING_MODE=dynamic
+REMME_PEERS_LIST=
```

Comment:
add an example of a value? 
Ex. `REMME_PEERS_LIST=tcp://hostname:port,tcp://hostname:port`

Reply:
added